sigstore/_internal/tuf.py

```diff
@@ -152,7 +153,12 @@ def _get(self, usage: str, statuses: list[str]) -> list[bytes]:
                 if path is None:
                     path = self._updater.download_target(target_info)
                 with open(path, "rb") as f:
-                    data.append(f.read())
+                    target_hash = f.read()
```

Comment:
Where does the `target_hash` variable name come from? If I'm reading this right this variable actually holds the TUF targets' raw contents, not a hash.

Reply:
If you refer to my idea in the original issue:

> the simplest thing might be for the TrustUpdater to log the hash (and maybe even the cached file location) of the artifacts

If that seems like a good idea, then you can just access `target_info`: it is of type [TargetFile](https://theupdateframework.readthedocs.io/en/latest/api/tuf.api.metadata.supporting.html#tuf.api.metadata.TargetFile) -- maybe something like `f"Updated {target_info.path}: {target_info.hashes}"` would work as a log line?

Reply:
LGTM -- although I don't remember if all targets are text files... if they are not, then the decode may fail

Reply:
I believe they all are, although maybe that's a bad future-looking assumption for us to make. Maybe we should limit debug logging to targets that should always be text?

Reply:
Yeah, all TUF targets are (conceptually) "files": a target is just a name and an associated bag of bytes.

The real underlying issue here (I think) is that not all TUF targets are guaranteed to be text; some might be binary instead. But that shouldn't be an issue with anything we currently attempt to retrieve for Sigstore purposes.